project.clj

```diff
@@ -21,5 +21,7 @@
                  [ring/ring-core "1.8.1"]
                  [ring/ring-json "0.5.0"]]
   :main clovertonemusic.core
+  :jar-exclusions [#"^public\/"]
+  :uberjar-exclusions [#"^public\/"]
```

Comment:
Running from the uberjar, the `/application.css` is not served, which breaks the whole site. I'm going to remove these lines.